src/dashboard.js

```diff
@@ -457,10 +446,10 @@ Telemetry.init(function () {
   
   // Add series button
   $("#addVersionButton").click(function () {
-    var state = null;
+    var state = "nightly/40/SIMPLE_MEASURES_FIRSTPAINT/saved_session/Firefox";
```

Comment:
It's the state that the first filter added to the dashboard will have - it's technically not allowed to happen in the UI, but if the cookie or URL hash is malformed or from an outdated version then this ensures that we don't simply stop showing filters like before.